assets/src/edit-story/components/carousel/zoomSelector/zoomSelector.js

```diff
@@ -43,32 +43,52 @@ const StyledDropDown = styled(DropDown)`
 const ZOOM_OPTIONS = [
   {
     label: _x('100%', 'zoom level', 'web-stories'),
-    value: ZOOM_SETTING.SINGLE,
+    value: 1,
   },
   {
     label: _x('200%', 'zoom level', 'web-stories'),
-    value: ZOOM_SETTING.DOUBLE,
+    value: 2,
   },
   { label: _x('Fill', 'zoom level', 'web-stories'), value: ZOOM_SETTING.FILL },
   { label: _x('Fit', 'zoom level', 'web-stories'), value: ZOOM_SETTING.FIT },
 ];
 
 function ZoomSelector() {
-  const { zoomSetting, setZoomSetting } = useLayout(
-    ({ state: { zoomSetting }, actions: { setZoomSetting } }) => ({
+  const { zoomSetting, zoomLevel, setZoomSetting, setZoomLevel } = useLayout(
+    ({
+      state: { zoomSetting, zoomLevel },
+      actions: { setZoomSetting, setZoomLevel },
+    }) => ({
       zoomSetting,
+      zoomLevel,
       setZoomSetting,
+      setZoomLevel,
     })
   );
 
-  const placeholder = useMemo(
-    () => ZOOM_OPTIONS.find(({ value }) => value === zoomSetting).label,
-    [zoomSetting]
-  );
+  const placeholder = useMemo(() => {
+    const option = ZOOM_OPTIONS.find(({ value }) => {
+      if (zoomSetting === ZOOM_SETTING.FIXED) {
+        return value === zoomLevel;
+      }
+      return value === zoomSetting;
+    });
+    if (option) {
+      return option.label;
+    }
+    return `${Math.round(zoomLevel * 100)}%`;
```

Comment:
@swissspidy, doesn't this need translation? Might there be a language, where this would be presented in a completely different way - like _"% 75"_ or whatever?

Reply:
It does! Different locales use different symbols for the percent sign, and it can be displayed on the left in some locales as well.

This should use `sprintf` and `_x()` with `zoom level` as the context (just like the strings above)